src/main/java/sc/fiji/bdvpg/services/ISourceAndConverterService.java

```diff
@@ -96,6 +96,6 @@
     /**
      * Finds the corresponding registered sac for a source.
      */
-    SourceAndConverter getSourceAndConverter( Source source );
+    SourceAndConverter getSourceAndConverterFromSource( Source source );
```

Comment:
Is this function new ? I don't remember where it comes from, but since you can have multiple SourceAndConverter for an identical Source, we should remove it and keep only the function `getSourceAndConvertersOfSource`.

Reply:
cf COmment, this should return a List<SourceAndConverter>